src/main/java/com/tteam/movieland/controller/MovieController.java

```diff
@@ -6,21 +6,27 @@
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
 
 @RestController
-@RequestMapping(value = "/api/v1", produces = MediaType.APPLICATION_JSON_VALUE)
+@RequestMapping(value = "/api/v1/movies", produces = MediaType.APPLICATION_JSON_VALUE)
 @RequiredArgsConstructor
 public class MovieController {
 
     private final MovieService movieService;
 
-    @GetMapping("movies")
-    protected List<Movie> getAllMovie() {
+    @GetMapping
+    protected @ResponseBody List<Movie> getAllMovie() {
```

Comment:
We should annotate on class "RestController" and methods without annotations "ResponseBody", or annotate on class "Controller" and each method mark on class "ResponseBody" annotation.

Reply:
Ok, fixed